src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -62,6 +64,50 @@ std::string compoundNumericPlugRepr( const T *plug )
 template<typename T>
 class CompoundNumericPlugSerialiser : public ValuePlugSerialiser
 {
+  public:
+
+	static std::string repr( const Gaffer::ValuePlug *plug, unsigned flagsMask, const std::string &extraArguments, const Serialisation *serialisation )
+	{
+		std::string result = ValuePlugSerialiser::repr( plug, flagsMask, extraArguments, serialisation );
+		if( const V3fPlug *v3fPlug = static_cast<const V3fPlug *>( plug ))
+		{
+			std::string interpretation;
+			switch( v3fPlug->getInterpretation() )
```

Comment:
So, I've tried doing this by calling `IECorePython::repr` like so:

`
	std::string extraArgs = IECorePython::repr( interpretation );
`

I get a linker error suggesting that we don't have a repr() for `GeometricData::Interpretation` and indeed we couldn't find one. Am I doing it wrong or do we just need to define that `repr()`?